patches/extra/ungoogled-chromium/add-flag-to-scroll-tabs.patch

```diff
@@ -38,11 +38,12 @@
  
  BrowserRootView::~BrowserRootView() {
    // It's possible to destroy the browser while a drop is active.  In this case,
-@@ -259,7 +274,7 @@ const char* BrowserRootView::GetClassNam
- }
+@@ -265,8 +280,7 @@ bool BrowserRootView::OnMouseWheel(const
  
- bool BrowserRootView::OnMouseWheel(const ui::MouseWheelEvent& event) {
--  if (browser_defaults::kScrollEventChangesTab) {
+   // Scroll-event-changes-tab is incompatible with scrolling tabstrip, so
+   // disable it if the latter feature is enabled.
+-  if (browser_defaults::kScrollEventChangesTab &&
+-      !base::FeatureList::IsEnabled(features::kScrollableTabStrip)) {
```

Comment:
It makes sense that `scroll_event_changes_tab_` takes the place of `browser_defaults::kScrollEventChangesTab`, but it is less clear to me whether it should also take the place of `!base::FeatureList::IsEnabled(features::kScrollableTabStrip)`, considering the comment above.

Reply:
Should I just leave `!base::FeatureList::IsEnabled(features::kScrollableTabStrip)` where it was then?

Reply:
Yes, unless you have a reason to remove it.

Reply:
I just pushed a commit to resolve this.